service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceGceInstanceConnectedTest.java

```diff
@@ -735,7 +735,7 @@ void deleteGceInstanceDo() throws Exception {
         FlightDebugInfo.newBuilder().doStepFailures(retrySteps).build());
 
     controlledResourceService.deleteControlledResourceSync(
-        resource.getWorkspaceId(), resource.getResourceId(), user.getAuthenticatedRequest());
+        resource.getWorkspaceId(), resource.getResourceId(), false, user.getAuthenticatedRequest());
```

Comment:
this should be true.

we are force deleting the instance aren't we?

even though we are not using this variable really in the flight.

Reply:
The only force applies to AWS notebooks, I think. It is irrelevant in this case.
Maybe we should have made an overload that had the old signature and filled in `false`.  Oh well